bedrock/firefox/templates/firefox/developer.html

```diff
@@ -57,14 +52,12 @@
   </h2>
   {{ download_firefox('alpha', icon=False, small=True, simple=True) }}
 
-  {% if l10n_has_tag('new_doorhanger') %}
   <p class="feedback-note">
     {{ _('Firefox Developer Edition automatically sends feedback to Mozilla.') }}
     <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
   </p>
-  {% endif %}
 
-  <img src="{{ static('img/firefox/developer/screenshot.jpg') }}" alt="{{ _('Firefox Developer Edition') }}" class="screenshot" />
+  {{ high_res_img('firefox/developer/challenger-hero.png', {'alt': 'Firefox Developer Edition', 'width': '832', 'height': 'auto', 'class': 'screenshot'}) }}
```

Comment:
Should this image also be behind the l10n / waffle flag as a conditional, otherwise some locales might see the screenshot image but not have the challenge listed in the links below?